test/cluster_test.go

```diff
@@ -168,8 +167,9 @@ func TestAuthServers(t *testing.T) {
 		t.Fatalf("Expect Auth failure, got no error\n")
 	}
 
-	if matched, _ := regexp.Match(`authorization`, []byte(err.Error())); !matched {
-		t.Fatalf("Wrong error, wanted Auth failure, got '%s'\n", err)
+	expected := `authorization`
```

Comment:
I fixed this one too in drain branch ;)

Reply:
PR to include fixes into the `drain` branch instead is here: https://github.com/nats-io/go-nats/pull/377 
closing this one for now